### ISSUE TITLE
Apk creation

### ISSUE THREAD

**Issue by 96Manuel** [react-native-device-info/react-native-device-info#1418]
Apk export

While exporting the android studio apk it gives me the following error:

Type com.learnium.RNDeviceInfo.BuildConfig is defined multiple times: C:\Users\manue\OneDrive\Desktop\Exentriq41\Exentriq-Talk\node_modules\react-native-device-info\android\build\.transforms\9166b1d6d34e6b3b03f3d84907b6a605\transformed\classes\classes.dex, C:\Users\manue\OneDrive\Desktop\Exentriq41\Exentriq-Talk\android\app\build\intermediates\external_libs_dex\release\mergeExtDexRelease\classes2.dex

I entered multiDexEnabled true

I followed this video to no avail: https://www.youtube.com/watch?v=aHHtTi2eVNs

But there is nothing to do keep returning this error, how can I fix it?

**Comment by mikehardy** [comment]
I don't believe multidex is required if you have a minSdkVersion of 21 or higher, and that has been the lowest supported version for 2 years now (https://github.com/facebook/react-native/commit/a17ff44adcf003dd4e4ef2301e1f80b77913f712) so it is unlikely you still need it, and unlikely that will fix the problem.

Something is going on that is project specific, and I'm not sure what but perhaps `npx react-native-clean-project` could fix it?

I haven't seen this before in this repo ever though, and don't reproduce it myself, and there's no reproduction here (https://stackoverflow.com/help/mcve)

If you can post a reproduction (https://stackoverflow.com/help/mcve) that triggers this we can certainly re-open but I'm going to have to close this as project-specific / not actionable in the meanwhile
